docs/website/blog/3.0.0-GA-blog.md

```diff
@@ -0,0 +1,333 @@
+? How do you want to bind the service? Bind As Files
+? Select naming strategy for binding names: DEFAULT
+ ✓  Successfully added the binding to the devfile.
+Run `odo dev` to create it on the cluster.
+You can automate this command by executing:
+  odo add binding --service cluster-sample.Cluster.postgresql.k8s.enterprisedb.io --name my-node-app-cluster-sample
+```
+
+_How is it different from `odo link` command of v2?_
+* [Interactive mode](#interactive-mode)
+* [Ability to connect to a service in any namespace](#ability-to-connect-to-a-service-in-any-namespace)
+* [Run from anywhere](#run-from-anywhere)
+* [Work without a running application](#work-without-a-running-application)
+* [Only supports Bindable Operators](#only-supports-bindable-operators)
+* [Requires the Service Binding Operator](#requires-the-service-binding-operator)
+
+#### Interactive mode
+`odo add binding` provides a very nice interactive mode that lists bindable service across all the namespaces, and provides a way to personalize related configuration, making the experience nicer.
+
+#### Ability to connect to a service in any namespace
+`odo add binding` allows the application to connect to any bindable service in any namespace, it does not necessarily have to be in the same namespace as the application.
+
+#### Run from anywhere
+`odo add binding` can be run from anywhere and does not require access to a Devfile in order to work. With this method, you can either directly create the Service Binding on the cluster, write the YAML to a file, or simply display it.
+
+#### Work without a running application
+As opposed to v2, `odo add binding` does not require for the application be running on the cluster, making it even faster.
+
+#### Only supports Bindable Operators
+
+`odo link` could link an application to another application and any Operator Backed service, but as of now, v3 only supports connecting to services that are backed by Operators considered [bindable](https://github.com/redhat-developer/service-binding-operator#known-bindable-operators) by the Service Binding Operator.
+
+Based on the user feedback, we might work on adding these features back, but as of now there are no plans of implementing them.
+
+Regardless, there are workarounds to connect to another application
+or non-bindable service, read [this blog](https://odo.dev/blog/binding-external-service-with-odo-v3) to know how.
+
+#### Requires the Service Binding Operator
+
+`odo add binding` requires that the Service Binding Operator is installed on the cluster before it is run. The command relies on the Operator to provide it with a list of Bindable services.
+
+However, this will be fixed in the future releases.
+
+### odo deploy
+
+`odo deploy` allows you to run the outerloop of your development cycle.
+Once you are satisfied with your application development locally, and are now ready to show your application to the world, you can use `odo deploy` to move to the next stage.
```

Comment:
Same comment as above. I couldn't think of better words, so I would appreciate some comments here as well.